common/scala/src/main/scala/whisk/core/entity/Identity.scala

```diff
@@ -38,13 +37,17 @@ object UserLimits extends DefaultJsonProtocol {
   implicit val serdes = jsonFormat3(UserLimits.apply)
 }
 
+protected[core] case class Namespace(name: EntityName, uuid: UUID)
```

Comment:
Do we need the nesting vs: add uuid to Identity and replace authkey with secret then?

Reply:
please see my comment above